tests/target/issue-189.rs

```diff
@@ -0,0 +1,14 @@
+// rustfmt-style_edition: 2024
+
+impl SomeType {
+    fn method(&mut self) {
+        self.array[array_index as usize]
+            .as_mut()
+            .expect("thing must exist")
+            .extra_info = Some(ExtraInfo {
+                parent,
+                count: count as u16,
+                children: children.into_boxed_slice(),
+            });
+    }
+}
```

Comment:
@johnhuichen can you open up a separate PR for this. These two issue are unrelated, right? Also, can you rename the file to reflect that this isn't rustfmt issue number 189, but rather https://github.com/rust-lang/style-team/issues/189

Reply:
Hi @ytmimi thanks for reviewing. Yes I can separate the two issues